src/index.ts

```diff
@@ -43,18 +59,32 @@ function getFileName(path: string, file: StrapiFile) {
 }
 
 function makeBlobServiceClient(config: Config) {
-    const account = trimParam(config.account);
-    const accountKey = trimParam(config.accountKey);
-    const sasToken = trimParam(config.sasToken);
     const serviceBaseURL = getServiceBaseUrl(config);
-    // if accountKey doesn't contain value return below line
-    if (sasToken != '') {
-        const anonymousCredential = new AnonymousCredential();
-        return new BlobServiceClient(`${serviceBaseURL}${sasToken}`, anonymousCredential);
+
+    switch (config.auth_type) {
```

Comment:
all config vars are camelCase. Will you update to the var name to `authType` thanks

Reply:
Right, will change

Reply:
Amended

Reply:
Thank you.